src/vsr/replica.zig

```diff
@@ -2396,7 +2396,7 @@ pub fn ReplicaType(
         fn on_sync_trailer(
             self: *Self,
             comptime trailer: vsr.SuperBlockTrailer,
-            message: *Message,
+            message: *const Message,
```

Comment:
`*const Message` means "can't manipulate reference count", but it actually still is possible to mutate the body of the message, right? In situations where we do mutate the body, but don't touch rc, do we use `*const Message` or `*Message`? I'd say `const` --- language says that `const`=shallowly immutable, and assigning any extra sematics would be error prone.